client-spark/spark3/src/test/java/org/apache/spark/shuffle/DelegationRssShuffleManagerTest.java

```diff
@@ -135,6 +137,31 @@ public void testCreateFallback() throws Exception {
     assertTrue(hasException);
   }
 
+  @Test
+  public void testTryAccessCluster() {
+    SparkConf conf = new SparkConf();
```

Comment:
Could we mock coordinator client? Could we use the method `tryAccessCluster` in this case?

Reply:
I don't think so, because `tryAccessCluster` is not directly called by the `CoordinatorClient`, and the `tryAccessCluster`method depends on the list of global object `CoordinatorClient`, so I don't have a better way for the time being.

Reply:
`coordinatorClients` is created by `RssSparkShuffleUtils.createCoordinatorClients(sparkConf)`. Could we imitate https://github.com/apache/incubator-uniffle/blob/79804c544b560ae3e872964a428d328dd71489a7/client-spark/spark2/src/test/java/org/apache/spark/shuffle/DelegationRssShuffleManagerTest.java#L79

Reply:
Got it.